src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -26,36 +26,36 @@
         <value>[base64 mime encoded string representing a byte array form of the .NET Framework object]</value>
         <comment>This is a comment</comment>
     </data>
-
-    There are any number of "resheader" rows that contain simple
+                

```

Comment:
Is there a reason for all of the trailing whitespace in this file?

Reply:
This whole file gets changed by Visual Studio _on the regular_. It rewrites this comment to add and remove whitespace depending on the time of day, phase of moon, price of socks, etc.

Reply:
It is also true that Visual Studio simply re-adds this comment with whatever broken spelling/grammar/punctuation it wants. Sorry. :(

Reply:
Most of the resws is autogenerated or auto-rewritten by Visual Studio, so that probably has something to do with it.